app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -258,7 +258,7 @@ protected void initializeWidgetPlacement(WidgetPlacement aPlacement) {
         aPlacement.width = windowWidth + mBorderWidth * 2;
         aPlacement.height = SettingsStore.getInstance(getContext()).getWindowHeight() + mBorderWidth * 2;
         aPlacement.worldWidth = WidgetPlacement.floatDimension(getContext(), R.dimen.window_world_width) *
-                (float)windowWidth / (float)SettingsStore.WINDOW_WIDTH_DEFAULT;
+                (float)windowWidth / (float)windowWidth;
```

Comment:
This change is not needed. The default size of the window in the 3D world dimensions (`R.dimen.window_world_width`) is calculated taking into account the default size of that window measured in dp (`SettingsStore.WINDOW_WIDTH_DEFAULT`).